Clean/Utils/Tactics/CircuitProofStart.lean

```diff
@@ -0,0 +1,93 @@
+import Lean
+import Clean.Circuit.Basic
+import Clean.Utils.Tactics.ProvableSimp
+
+open Lean Elab Tactic Meta
+open Circuit
+
+/--
+  Apply intro until we've introduced all the standard parameters for
+  soundness or completeness proofs, then apply additional simplifications
```

Comment:
Better phrasing is:

Introduce all standard parameters and hypotheses for Soundness or Completeness.